R/mod_scrollytell_example.R

```diff
@@ -111,8 +126,8 @@ mod_scrollytell_example_server <- function(id){
       # create a custom chart dataset based on the scrolly section inputs
       # the input$scroll_level will allow you to define the chart input
       # this input is based on section of the report that is currently active during the scroll
-      chart_data <- iris |>
-        dplyr::filter(Species %in% switch(input$scroll_level,
+      chart_data <- datasets::iris |>
+        dplyr::filter(.data$Species %in% switch(input$scroll_level,
```

Comment:
Best practice to use the `.data` pronoun for variables in things like `data.frames`. It silences R CMD check issues.